source_gen/pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
   sdk: ">=2.1.0 <3.0.0"
```

Comment:
Do we need to increase the lower bound here?

Reply:
Why?
`package:analyzer` does have a higher lower bound `>=2.2.2 <3.0.0`.
But I don't know what is you motivation to increase.